src/master-playlist-controller.js

```diff
@@ -505,7 +532,7 @@ export default class MasterPlaylistController extends videojs.EventTarget {
    * making it unavailable for selection by the rendition selection algorithm
    * and then forces a new playlist (rendition) selection.
    */
-  blacklistCurrentPlaylist_(error) {
+  blacklistCurrentPlaylist(error = {}) {
```

Comment:
Basically so that we can call `blacklistCurrentPlaylist` without passing an error which we only used for the `error.playlist` value anyway. I can switch it back to not using the default but then I will just be doing the same thing the code does when it finds that error has no playlist:  `currentPlaylist = error.playlist || this.masterPlaylistLoader_.media();`